server_addon/houdini/client/ayon_houdini/plugins/create/create_hda.py

```diff
@@ -56,8 +60,18 @@ def create_instance_node(
                 raise CreatorError(
                     "cannot create hda from node {}".format(to_hda))
 
+            # Pick a unique type name for HDA product per folder path per project.
+            type_name = (
+                "{project_name}{folder_path}_{node_name}".format(
+                    project_name=get_current_project_name(),
+                    folder_path=get_current_folder_path().replace("/","_"),
```

Comment:
Why not use the `folder_path` argument that is being passed into this function instead of the current one?

Reply:
I didn't notice it 🤦‍♂️. let me fix it.

Reply:
Thank you
https://github.com/ynput/ayon-core/pull/644/commits/f464d190321d4c2289eb92c98c95eeb44eb0de53